pkg/apis/openebs.io/v1alpha1/cas_volume_metrics.go

```diff
@@ -18,7 +18,9 @@ package v1alpha1
 
 // VolumeMetrics is used to store the collected metrics
 // all the stats exposed by jiva stored into OpenEBSVolumeMetrics fields
-type VolumeMetrics []struct {
+type VolumeMetrics []VolumeMetric
```

Comment:
Can we rename to `VolumeMetricList` to keep the naming conventions same

Reply:
done